Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,209 @@
+          pradeo.device.imei: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.imei}}"
+          pradeo.device.byod: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.byod}}"
+          pradeo.device.declaredOperatingSystem: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.declaredOperatingSystem}}"
+          pradeo.device.declaredOperatingSystemVersion: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.declaredOperatingSystemVersion}}"
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.declaredOperatingSystemSecurityPatchDate}}"
+          pradeo.device.declaredModel: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.declaredModel}}"
+          pradeo.device.lastConnection: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.enrollmentStatus.lastConnection}}"
+          pradeo.device.coupled: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.enrollmentStatus.coupled}}"
+          pradeo.device.mdmId: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.emmDeviceInfo.externalId}}"
+          pradeo.device.emm: "{{json_event.message.content.deviceApplicationCompliance.deviceApplication.device.emmDeviceInfo.emm}}"
+          pradeo.compliance.matchedResponseRules: "{{json_event.message.content.deviceApplication.compliance.matchedResponseRules}}"
+          pradeo.application.id: "{{json_event.message.content.deviceApplicationCompliance.application.id}}"
+          pradeo.application.package: "{{json_event.message.content.deviceApplicationCompliance.application.package.package}}"
+          pradeo.application.system: "{{json_event.message.content.deviceApplicationCompliance.application.package.system}}"
+          pradeo.application.version: "{{json_event.message.content.deviceApplicationCompliance.application.version}}"
+          pradeo.application.versionCode: "{{json_event.message.content.deviceApplicationCompliance.application.versionCode}}"
+          pradeo.application.name: "{{json_event.message.content.deviceApplicationCompliance.application.name}}"
+          pradeo.application.md5: "{{json_event.message.content.deviceApplicationCompliance.application.md5}}"
+          pradeo.application.sha1: "{{json_event.message.content.deviceApplicationCompliance.application.sha1}}"
+          pradeo.application.sha256: "{{json_event.message.content.deviceApplicationCompliance.application.sha256}}"
+          pradeo.detection.status: "{{json_event.message.content.deviceApplicationCompliance.status}}"
+      - filter: '{{json_event.message.type == "DeviceComplianceUpdated"}}'
+        set:
+          event.category: "process"
+          event.type: "change"
```

These fields must be arrays
```suggestion
          event.category: ["process"]
          event.type: ["change"]
```